LEGO1/mxmusicpresenter.cpp

```diff
@@ -21,4 +27,22 @@ void MxMusicPresenter::Init()
 void MxMusicPresenter::Destroy(MxBool)
 {
   // TODO
-}
\ No newline at end of file
+}
+
+// OFFSET: LEGO1 0x100c25a0
+MxResult MxMusicPresenter::AddToMusicManager()
+{
+  MxResult ret = -1;
+  if(MusicManager()) {
+    ret = 0;
+    MusicManager()->AddPresenter(*this);
+  }
+  return ret;
+} 
+
+// OFFSET: LEGO1 0x100c25d0
+void MxMusicPresenter::vtable38()
+{
+  // TODO: Name this function when we know what the argument to Destroy does
+  Destroy(FALSE);
```

Comment:
Looking at 0x100b55fe, if that argument is false, then MxPresenter::Init is called

Reply:
Cool! I'd implement that now but I'd rather make another PR for Destroy for MxMusicPresenter since it requires making a stub and also making a member public in MxPresenter